main/webapp/modules/core/scripts/views/data-table/cell-ui.js

```diff
@@ -122,7 +122,7 @@ DataTableCellUI.prototype._render = function() {
       }
       if (spanArr.length != 0) {
         var span = document.createElement('span');
-        span.textContent = spanArr.join(" ");
+        span.innerHTML = spanArr.join(" ").replace(/\n/, '<br />');
```

Comment:
If cells contain HTML markup, then this would interpret the HTML markup directly by embedding the elements in OpenRefine, which is not what we want, I think :)

Reply:
Updated. Please check @wetneb